ethereum-common/src/utils.rs

```diff
@@ -34,43 +38,51 @@ impl fmt::Display for MerkleProofError {
 
 impl core::error::Error for MerkleProofError {}
 
+/// Contains Merkle proof for the provided receipt.
 pub struct MerkleProof {
     pub proof: Vec<Vec<u8>>,
```

Comment:
Are there some specs on the format of this merkle proof in the Ethereum docs?

Reply:
Frankly speaking I didn't search for that since we're using `trie-db` crate to generate/verify Merkle-proofs.